contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_list.handlebars

```diff
@@ -0,0 +1,30 @@
+<div id="clipboard" class="pull-right">
+	{{#if collapsed}}
+		<button class="btn btn-default clipboard-toggler clipboard-button" type="button">
+			<span class="glyphicon glyphicon-menu-down" aria-hidden="true"></span>
+			Clipboard
+			<span class="badge"></span>
+		</button>
+	{{else}}
+		<div id="main-clipboard">
+			<div class="clipboard_header">
+				<h4>
+					<span class="clipboard-toggler glyphicon glyphicon-menu-up" aria-hidden="true"></span>
+					Clipboard
+				</h4>
+			</div>
+			<div id="clipboard_content">
+				<div id = "list" class="list_content">
+					<div id="list_background"></div>
+					<ul class="list-unstyled content-list">
+						{{#each content_list}}
+							<li id="clipboard_item_{{id}}" class="clipboard_item"></li>
```

Comment:
No need to address in this PR, but it would be better if each item was a separate view, so that we can say, edit the metadata of an item, save the change, and rerender that item, without having to rerender the entire list.

Think of how we can reuse code from the topic tree view for this and the trash view as well.